src/sentry/static/sentry/app/components/similarScoreCard.tsx

```diff
@@ -22,15 +29,37 @@ const SimilarScoreCard = ({scoreList = []}: Props) => {
     return null;
   }
 
+  let sumOtherScores = 0;
+  let numOtherScores = 0;
+
   return (
     <React.Fragment>
-      {scoreList.map(([key, score]) => (
-        <Wrapper key={key}>
-          <div>{scoreComponents[key]}</div>
+      {scoreList.map(([key, score]) => {
+        const title =
+          scoreComponents[key.replace(/similarity:\d\d\d\d-\d\d-\d\d/, 'similarity:*')];
```

Comment:
This should not happen. The keys of array `scoreList` should be one of the object `scoreComponents`

Reply:
should we perhaps move it to `similarStackTrace`?

Reply:
Ah, yes. I'll update the types.